authors/apps/articles/views.py

```diff
@@ -180,12 +175,12 @@ def post(self, request, **kwargs):
             status=status.HTTP_200_OK,
         )
 
-    @swagger_auto_schema(
-        operation_description='Unfavorite an Article.',
-        operation_id='unfavorite an article',
-        request_body=None,
-        responses={200: serializer_class(many=False), 400: 'BAD REQUEST'},
-    )
+    # @swagger_auto_schema(
```

Comment:
Good work @neelxie , please remove this commented out code

Reply:
I have removed the commented code and re-pushed.